test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -175,6 +178,23 @@ export default class PerfCompare extends Component {
         })
     }
 
+    async getChildrenList ( resBenchmarkBuildJson ) {
+        const childList = [];
+        if ( resBenchmarkBuildJson.testInfo[0].hasChildren === true ) {
+            let parentIdBuild = resBenchmarkBuildJson.testInfo[0]._id;
+            let childListBuild = await fetch( `/api/getChildBuilds?parentId=${parentIdBuild}`, {
+                method: 'get'
+            } );
+            let childListBuildJson = await childListBuild.json();
+            for ( let i = 0; i < childListBuildJson.length; i++ ) {
+                if ( childListBuildJson[i].tests && childListBuildJson[i].tests.length > 0 ) {
+                    childList.push (childListBuildJson[i].tests[0].testData.metrics);
```

Comment:
Need to take care of the scenario when we're running more than one test in one Jenkins job.